WalletWasabi.Tests/Helpers/ServiceFactory.cs

```diff
@@ -40,8 +41,8 @@ public static class ServiceFactory
 				}
 			}
 			var coinsView = new CoinsView(scoins);
-			var transactionStore = new AllTransactionStoreMock(workFolderPath: ".", Network.Main);
-			return new TransactionFactory(Network.Main, keyManager, coinsView, transactionStore, password, allowUnconfirmed);
+			var mockTransactionStore = new Mock<AllTransactionStore>(".", Network.Main);
```

Comment:
I would also consider using `MockBehavior.Strict`: https://github.com/Moq/moq4/wiki/Quickstart#customizing-mock-behavior

Reply:
No. That would force us to setup the mock. It could be considered to use CallBase instead but it seems this mock is only to fill a gap.